Cofense intelligence v2 integration (#12671)
demisto/content: Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/README.md

* *Cofense.URL.Malicious.Description* - this output was replaced by *CofenseIntelligence.URL.Malicious.Description*.
* *Cofense.URL.Cofense.ThreatIDs* - this output was replaced by *CofenseIntelligence.URL.Cofense.ThreatIDs*.

In the *file* command:
* *Cofense.File.MD5* - this output was replaced by *CofenseIntelligence.File.MD5*.
* *Cofense.File.Malicious.Vendor* - this output was replaced by *CofenseIntelligence.File.Malicious.Vendor*.
* *Cofense.File.Malicious.Description* - this output was replaced by *CofenseIntelligence.File.Malicious.Description*.
* *Cofense.File.ThreatIDs* - this output was replaced by *CofenseIntelligence.File.ThreatIDs*.

In the *ip* command:
* *Cofense.IP.Data* - this output was replaced by *CofenseIntelligence.IP.Data*.
* *Cofense.IP.Malicious.Vendor* - this output was replaced by *CofenseIntelligence.IP.Malicious.Vendor*.
* *Cofense.IP.Malicious.Description* - this output was replaced by *CofenseIntelligence.IP.Malicious.Description*.
* *Cofense.IP.Cofense.ThreatIDs* - this output was replaced by *CofenseIntelligence.IP.Cofense.ThreatIDs*.

In the *email* command:

> * *Account.Email.Malicious.Vendor*

Reviewer: ```suggestion
* *Account.Email.Malicious.Vendor* - this output was replaced by *CofenseIntelligence.Email.Malicious.Vendor*.
```

> * *Account.Email.Malicious.Description*

Reviewer: ```suggestion
* *Account.Email.Malicious.Description* - this output was replaced by *CofenseIntelligence.Email.Malicious.Description*.
```

Use the Cofense Intelligence integration to check the reputation of URLs, IP addresses, file hashes, and email addresses.

> This integration was integrated and tested with version 02 of CofenseIntelligenceV2

Reviewer: ```suggestion
This integration was integrated and tested with version 02 of Cofense Intelligence
```

Search for threats associated with an indicator.

> The verdict (Unknown, Benign, Suspicious, Malicious) of each threat is determined by the impact (None, Minor, Moderate, Major) of its associated web locations as detected in cofense ,  along with a threshold value that is being set by the user (when configuring the instance):

Reviewer: ```suggestion
The verdict (Unknown, Benign, Suspicious, Malicious) of each threat is determined by the impact (None, Minor, Moderate, Major) of its associated web locations as detected in cofense,  along with a threshold value that is being set by the user (when configuring the instance):
```

For each Threat: Verdict = if indicator_found: the_impact_of_the_indicator, 

> else max ( list of all the related indicators’ impact)

Reviewer: maybe we should change this to a quick paragraph for those uninitiated in if-else statments.

| CofenseIntelligence.IP.Threats.hasReport | Boolean | Flag to show whether this campaign has a written report associated with it | 
| CofenseIntelligence.IP.Threats.reportURL | String | Direct URL to human readable report for this campaign | 
| CofenseIntelligence.IP.Threats.apiReportURL | String | URL to human readable report for this campaign | 
| CofenseIntelligence.IP.Threats.threatDetailURL | String | T3 Report URL | 
| CofenseIntelligence.IP.Threats.malwareFamilySet.familyName | String | Family name of malware | 
| CofenseIntelligence.IP.Threats.malwareFamilySet.description | String | Set of malware family | 
| CofenseIntelligence.IP.Threats.threatType | String | This will only have one value for malware | 

#### Command Example
```!ip ip=8.8.8.8 using=CofenseIntelligenceV2_instance_1```

#### Context Example
```json
{
    "DBotScore": {
        "Indicator": "8.8.8.8",
        "Reliability": "B - Usually reliable",
        "Score": 0,
        "Type": "ip",
        "Vendor": "CofenseIntelligenceV2"
    },
    "IP": {
        "Address": "8.8.8.8"
    }
}
```

#### Human Readable Output

>### Cofense IP Reputation for IP 8.8.8.8

> >**No entries.**

Reviewer: please give an example with entries.

>| 14794 | Adaptable financial crimes botnet trojan with email worm and malware delivery capabilities, also known as Emotet<br/>Advanced banking trojan utilizing both webinjects and browser redirection | Multilingual invoice themed messages deliver Microsoft Word documents containing Office macros which download Geodo malware. Geodo then downloads and runs the banking trojan IcedID. | Invoices - OfficeMacro, Geodo, IcedID | 2018-11-08 20:13:04 |

> >| 14943 | Information stealer designed to steal stored credentials and cryptocurrency wallets | Receipt themed emails deliver AZORult malware. | CTM Receipt - AZORult | 2018-11-13 15:55:51 |

Reviewer: lets trim is down to:
1. only 2 lines.
2. cut the text - you can make up your text here.

| CofenseIntelligence.File.Threats.hasReport | Boolean | Flag to show whether this campaign has a written report associated with it | 
| CofenseIntelligence.File.Threats.reportURL | String | Direct URL to human readable report for this campaign | 
| CofenseIntelligence.File.Threats.apiReportURL | String | URL to human readable report for this campaign | 
| CofenseIntelligence.File.Threats.threatDetailURL | String | T3 Report URL | 
| CofenseIntelligence.File.Threats.malwareFamilySet.familyName | String | Family name of malware | 
| CofenseIntelligence.File.Threats.malwareFamilySet.description | String | Set of malware family | 
| CofenseIntelligence.File.Threats.threatType | String | This will only have one value for malware | 

#### Command Example
```!file file=md5 using=CofenseIntelligenceV2_instance_1```

#### Context Example
```json
{
    "DBotScore": {
        "Indicator": "md5",
        "Reliability": "B - Usually reliable",
        "Score": 0,
        "Type": "file",
        "Vendor": "CofenseIntelligenceV2"
    },
    "File": {
        "MD5": "md5"
    }
}
```

#### Human Readable Output

>### Cofense file Reputation for file md5

> >**No entries.**

Reviewer: please show an example with entries.

| CofenseIntelligence.Email.Threats.reportURL | String | Direct URL to human readable report for this campaign | 
| CofenseIntelligence.Email.Threats.apiReportURL | String | URL to human readable report for this campaign | 
| CofenseIntelligence.Email.Threats.threatDetailURL | String | T3 Report URL | 
| CofenseIntelligence.Email.Threats.malwareFamilySet.familyName | String | Family name of malware | 
| CofenseIntelligence.Email.Threats.malwareFamilySet.description | String | Set of malware family | 
| CofenseIntelligence.Email.Threats.threatType | String | This will only have one value for malware | 

#### Command Example
```!email email=email@email.com using=CofenseIntelligenceV2_instance_1```

#### Context Example
```json
{
    "DBotScore": {
        "Indicator": "email@email.com",
        "Reliability": "B - Usually reliable",
        "Score": 0,
        "Type": "email",
        "Vendor": "CofenseIntelligenceV2"
    },
    "EMAIL": {
        "Address": "email@email.com",
        "Domain": "email.com"
    }
}
```

#### Human Readable Output

>### Cofense email Reputation for email email@email.com

> >**No entries.**

Reviewer: please show an example with entries.

                            "threatVendorName": "TrendMicro"
                        },
                        {
                            "detected": false,
                            "threatVendorName": "Arcabit"
                        },
                        {
                            "detected": false,
                            "threatVendorName": "BitDefender"
                        },
                        {
                            "detected": true,
                            "threatVendorName": "Kaspersky"
                        },
                        {
                            "detected": false,
                            "threatVendorName": "CAT-QuickHeal"
                        },
                        {
                            "detected": false,
                            "threatVendorName": "F-Secure"
                        },
                        {
                            "detected": true,
                            "threatVendorName": "ZoneAlarm"
                        },
                        {
                            "detected": false,
                            "threatVendorName": "Yandex"

>                         }

Reviewer: trim this down - there is no need to show all the info.

            "relatedSearchTags": [],
            "reportURL": "https://www.threathq.com/api/l/activethreatreport/10882/html",
            "senderEmailSet": [],
            "senderIpSet": [],
            "senderNameSet": [],
            "spamUrlSet": [],
            "subjectSet": [
                {
                    "subject": "Re-confirm your banking details",
                    "totalCount": 1
                }
            ],
            "threatDetailURL": "https://www.threathq.com/p42/search/default?m=10882",
            "threatType": "MALWARE"
        }
    },
    "DBotScore": {
        "Indicator": "http://prosciuttuamo.it/tmp/ifeanyi.exe",
        "Reliability": "B - Usually reliable",
        "Score": 3,
        "Type": "url",
        "Vendor": "CofenseIntelligenceV2"
    },
    "URL": {
        "Data": "http://prosciuttuamo.it/tmp/ifeanyi.exe",
        "Malicious": {
            "Description": null,
            "Vendor": "CofenseIntelligenceV2"
        }
    }
}
```

#### Human Readable Output

>### Cofense URL Reputation for url http://prosciuttuamo.it/tmp/ifeanyi.exe
>|Threat ID|Threat Types|Verdict|Executive Summary|Campaign|Last Published|
>|---|---|---|---|---|---|

> >| 10882 | Agent Tesla collects sensitive information, such as saved credentials for web, ftp, email, and instant messaging clients. Additionally, Tesla gathers data about the victim's PC and captures keystrokes. | Bad | These emails claim to be a request to confirm banking details within the attached Word document. In reality, the campaign delivers a weaponised .doc file which contains obfuscated macros, written to download and execute a payload from a remote server. In this instance, the retrieved executable is a sample of the Agent Tesla keylogger. Upon execution, Agent Tesla starts collecting sensitive information, such as saved credentials for web, ftp, email, and instant messaging clients. Additionally, Tesla gathers data about the victim's PC and captures keystrokes. | Banking Details - OfficeMacro, Agent Tesla Keylogger | 2018-09-07 16:40:35 |

Reviewer: let's shorten the text here.

> 

Reviewer: talk to @yaakovpraisler and please add a section for the changes made from v1.
